crates/api_common/src/request.rs

```diff
@@ -96,7 +98,7 @@ fn extract_opengraph_data(html_bytes: &[u8], url: &Url) -> Result<OpenGraphData,
     .ok_or(LemmyErrorType::NoLinesInHtml)?
     .to_lowercase();
 
-  if !first_line.starts_with("<!doctype html>") {
+  if !first_line.starts_with("<!doctype html") {
     Err(LemmyErrorType::SiteMetadataPageIsNotDoctypeHtml)?
   }
```

Comment:
Just noticed this change after reading https://github.com/LemmyNet/lemmy/issues/4468. Do we actually need this check at all? Why not attempt `HTML::from_string` always and let it throw an error if its invalid? A check for max file size would make more sense.